cugraph-analytics/cugraph_analytics.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running Graph Analytics on Large Scale Graphs Effortlessly with Nvidia and Memgraph \n",
+    "\n",
+    "This tutorial will show you how to use  **PageRank** graph analysis and **Louvain** community detection on a **Facebook dataset** containing 1.3 million relationships. Upon completing it, you will know how to run analytics algorithms on your dataset using **Python**. Then, you can run any of the following algorithms:\n",
+    "* Balanced Cut (clustering)\n",
+    "* Spectral Clustering (clustering)\n",
+    "* HITS (hubs vs. authorities analytics)\n",
+    "* Leiden (community detection)\n",
+    "* Katz Centrality\n",
+    "* Betweenness Centrality\n",
+    "\n",
+    "All of the algorithms above are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "To follow the tutorial, please install:\n",
+    "- [Docker](https://docs.docker.com/get-docker/) - needed to run `mage-cugraph` image\n",
+    "- [Jupyter](https://jupyter.org/install) - using jupyter-notebook you can write  `CSV` importing and graph analytics in one file\n",
+    "- [GQLAlchemy](https://pypi.org/project/gqlalchemy/) - used to connect Memgraph with Python\n",
+    "- [Memgraph Lab](https://memgraph.com/lab) - a GUI tool we use to visualize graphs\n",
+    "\n",
+    "Here are brief instructions on how to install everything you will need."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Memgraph with Docker\n",
+    "We need *[Docker](https://www.docker.com/)* because Memgraph is a native Linux application and can't be installed on Windows and macOS. In this tutorial, we will use `mage-cugraph` Docker image. Check our [guide](https://memgraph.com/docs/mage) on setting everything up.\n",
+    "\n",
+    "\n",
+    "Before running `mage-cugraph` Docker image, position yourself inside `jupyter-memgraph-tutorials/cugraph-analytics` folder in terminal. We are ready to run the `mage-cugraph` image, so let's do a little \"hack\" in the docker run command that will be useful to us later.\n",
```

Comment:
Condense
```suggestion
    "Before running the `mage-cugraph` image, position yourself inside the `jupyter-memgraph-tutorials/cugraph-analytics` folder with the terminal.
```

Reply:
Do you think it makes sense to remove this explanation of "hack", why won't that be interesting to the user?

Reply:
Initially I removed it because it was a bit repetitive, but the following sentence works:

"Before running the `mage-cugraph` image, position yourself inside the `jupyter-memgraph-tutorials/cugraph-analytics` folder with the terminal. When you get there, we’ll show you a nifty \"hack\" with the `docker run` command."